app/controllers/documents_controller.rb

```diff
@@ -19,4 +19,10 @@ def letter_file_name(doc)
     letter_template = meta.dig(:template_id)
     pay_ref.to_s + '_' + letter_template.to_s + doc.extension
   end
+
+  def params_for_index
```

Comment:
SUGGEST: renaming this to something more relevant
```suggestion
  def payment_ref_from_params
  
```